ui/templates/base.html

```diff
@@ -7,6 +7,10 @@
     <script type="text/javascript">
     var SETTINGS = {{ js_settings_json|safe }};
     </script>
+    {% if style_public_src %}
+    <script type="text/javascript" src="{{ style_public_src }}"></script>
+    <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
```

Comment:
hrm, actually tried removing this and ran into some issues.

do you think it's fine to rely on the cdn version? it works fine if I just leave the cdn `script` tag up here and remove jquery from the `public.js` bundle.